CRM/Contact/BAO/GroupContactCache.php

```diff
@@ -216,7 +216,7 @@ public static function updateCacheTime($groupID, $processed) {
    * @param int $groupID
    *
    */
-  public static function clearGroupContactCache($groupID): void {
+  protected static function clearGroupContactCache($groupID): void {
```

Comment:
It looks like these were `static` before and after the change - maybe a typo in subject?

I guess the idea here is that actively *clearing* the cache is more destructive action than *invalidating* the cache? So the *clear* operation should be a more reserved/internal thing - and external callers should use the lighter *invalidate* operation.

Which sounds good to me, esp since it wasn't used in `universe`.

Reply:
@totten  - yep that was the wrong word!